koku/masu/util/ocp/ocp_post_processor.py

```diff
@@ -36,6 +36,9 @@ def process_openshift_labels(label_string):
     Returns:
         (dict): The JSON dictionary made from the label string
 
+    Dev Note:
+        You can use 'kube_node_labels` in prometheus to see the values.
+
```

Comment:
This is not a complete statement. There are 5 metrics we query to gather all the labels. Let's just remove this and we can reference the operator if we need to see what queries to run to see labels.